addons/website/static/src/builder/plugins/options/avatars_header_buttons.js

```diff
@@ -0,0 +1,33 @@
+import { Component } from "@odoo/owl";
+import { useDomState } from "@html_builder/core/utils";
+import { useOperation } from "@html_builder/core/operation_plugin";
+
+export class AvatarsHeaderMiddleButtons extends Component {
+    static template = "website.AvatarsHeaderMiddleButtons";
+    static props = {
+        addAvatar: Function,
+        removeAvatar: Function,
+    };
+
+    setup() {
+        this.callOperation = useOperation();
+        this.state = useDomState((editingElement) => {
+            const containerEl = editingElement.querySelector('.s_avatars_rules_collector');
+            return {
+                avatarEls: containerEl ? containerEl.querySelectorAll('.o_avatar') : [],
+            };
```

Comment:
As said in the xml file, return the status if it is disabled here, instead of giving the avatars list and then computing in the XML :slightly_smiling_face: 
```js
const avatarEls = editingElement.querySelectorAll(
    ".s_avatars_rules_collector .o_avatar"
);
return {
    disableRemoveButton: avatarEls.length <= 1,
};
```

Additionally, given that the `s_avatars_rules_collector` element is unremovable, I don't think there is a case where it would not be here so no need to do the `containerEl ?` check :slightly_smiling_face: That said, you can get the avatars directly with the `.s_avatars_rules_collector .o_avatar` selector without having to get the container first, it will indirectly do the check :wink: